src/apps/inventory/models.py

```diff
@@ -1,5 +1,7 @@
 from django.db import models
 
+from src.apps.user.models import User
```

Comment:
Import User via get_user_model()

when you create Foreign key on user model  it is better to use settings.AUTH_USER_MODEL